rj_gameplay/rj_gameplay/tactic/nmark_tactic.py

```diff
@@ -65,7 +65,7 @@ def __call__(
 
     def unassigned_cost_fn(
         self,
-        prev_result: Optional["RoleResult"],
+        prev_result: Optional[role.RoleResult],
```

Comment:
Error: rj_gameplay/rj_gameplay/tactic/nmark_tactic.py:68: error: Name 'RoleResult' is not defined
-> See Comment Above